source/reference/minio-mc-admin/mc-admin-heal.rst

```diff
@@ -60,37 +60,62 @@ Syntax
    If the ``TARGET`` bucket or bucket prefix has an active healing scan,
    the command returns the status of that scan.
 
-.. mc-cmd:: --scan
-   
 
-   The type of scan to perform. Specify one of the following supported scan
-   modes:
+++++++++++++++++++++
 
-   - ``normal`` (default)
-   - ``deep``
 
-.. mc-cmd:: --recursive, r
-   
+.. dropdown:: Deprecated Arguments
 
-   Recursively scans for objects in the specified bucket or bucket prefix.
+   The following command flags have been deprecated and should only be used under guidance from MinIO Engineers in association with a SUBNET ticket.
 
-.. mc-cmd:: --dry-run
-   
+   - ``--scan`` 
+     
+     The type of scan to perform. Specify one of the following supported scan modes:
 
-   Inspects the :mc-cmd:`~mc admin heal TARGET` bucket or bucket prefix, 
-   but does *not* perform any object healing.
+       - ``normal`` (default)
+       - ``deep``
 
-.. mc-cmd:: --force-start, f
-   
+   - ``--recursive, r`` 
+     
+     Recursively scans for objects in the specified bucket or bucket prefix.
 
-   Force starts the healing process.
+   - ``--dry-run`` 
+     
+     Inspects the :mc-cmd:`~mc admin heal TARGET` bucket or bucket prefix, but does *not* perform any object healing.
 
-.. mc-cmd:: --force-stop, s
-   
+   - ``--force-start, f`` 
+     
+     Force starts the healing process.
 
-   Force stops the healing sequence.
+   - ``--force-stop, s`` 
+     
+     Force stops the healing sequence.
 
-.. mc-cmd:: --remove
-   
+   - ``--remove`` 
+     
+     Removes dangling objects and data directories in the healing process not referenced by the metadata on a per-drive basis.
 
-   Removes dangling objects and data directories in the healing process not referenced by the metadata on a per-drive basis.
+Colors
+------
+
+Originally, the healing mechanism output a table that used colors to attempt to differentiate the status of objects in healing.
+These colors lack much useful detail and have been deprecated in favor of :ref:`healing metrics available at the cluster level <minio-metrics-and-alerts-available-metrics>`.
```

Comment:
:thinking: I think we're OK in simply stating that the the color-based output has been deprecated.

```suggestion
These have been deprecated in favor of more detailed :ref:`healing metrics available at the cluster level <minio-metrics-and-alerts-available-metrics>`.
```

Reply:
Yeah, I wondered about that even as I wrote it.
Taking your suggestion as is for now.